src/Controller.php

```diff
@@ -9,53 +9,39 @@
 use Symfony\Component\Console\Output\ConsoleOutputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
-final class Controller
+final class Controller implements ControllerInterface
 {
-    /**
-     * @var OutputInterface
-     */
-    private $output;
-    /**
-     * @var GlobalConfig
-     */
-    private $config;
 
-    public function __construct(OutputInterface $output, GlobalConfig $config)
+    public function ssh(OutputInterface $output, GlobalConfig $config, string $destination, string $currentHost, string $command, bool $dryRun): int
```

Comment:
why do want to make OutputInterface and GlobalConfig as a parameter for every method?